junebug/stores.py

```diff
@@ -0,0 +1,53 @@
+from twisted.internet.defer import inlineCallbacks, returnValue
+from vumi.message import TransportUserMessage
+
+
+class BaseStore(object):
+    '''Base class for store objects. Stores data in redis in a hash.
+    redis: redis manager
+    ttl: expiry for keys in the store
```

Comment:
Should we update this docstring to use the usual Sphinx formatting? That way if we ever pull it into autodoc it'll just work (plus it'll be more consistent with our docs elsewhere).